extensions/spark/src/main/scala/ai/djl/spark/tokenizer/TokenizerFactory.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ * with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+ * OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package ai.djl.spark.tokenizer
+
+import ai.djl.modality.nlp.preprocess.Tokenizer
+
+/**
+ * [[TokenizerFactory]] contains tokenizer creation mechanism on top of different platforms.
+ * System will choose appropriate Factory based on the supported audio type.
+ */
+class TokenizerFactory {
```

Comment:
Should we add TokenizerFactory in our api package?

Reply:
Maybe leave it here for now and refactor later

Reply:
Changed.